package.json

```diff
@@ -3,6 +3,9 @@
   "version": "1.0.0",
   "description": "Authors Haven Project Front End",
   "main": "index.js",
+  "engines": {
```

Comment:
Is there a reason why you are adding this property?

Reply:
this was just to specify the node version I am working with